### PR TITLE
update shap to 0.32.1

### PR DESCRIPTION
**Pull request by imatiach-msft**
update shap to 0.31.0 for avanade customer bugfix that exists in latest shap

edit: 0.32.1

### CODE REVIEW

**Comment by bethz** [comment]
All the testing that has been done has been done on the existing version of shap.
Do we want to tag for ignite and then add this in?

**Comment by imatiach-msft** [comment]
@bethz  agree this shouldn't go in for ignite.  We can make the next release of interpret-community after ignite.

**Comment by bethz** [comment]
Adding this in as soon as we make a final tag for ignite is fine too.

File: python/setup.py [hunk 1]
```diff
@@ -54,15 +54,17 @@
...
 ]
 
 EXTRAS = {
     'sample': [
         'hdbscan'
     ],
     'deep': [
-        'tensorflow'
+        'tensorflow',
+        'pyyaml',
```
Comment (wamartin-aml) [inline python/setup.py]: why do we need pyyaml?

Comment (imatiach-msft) [reply python/setup.py]: it caused errors on linux for keras

**Pull request merged**